kube/services/gearbox/gearbox-deploy.yaml

```diff
@@ -89,6 +95,10 @@ spec:
         ports:
         - containerPort: 80
         volumeMounts:
+          - name: "gearbox-jwt-keys"
+            readOnly: true
+            mountPath: "/src/src/gearbox/keys/jwt_public_key.pem"
```

Comment:
is the /src/src a typo or is it actually 2 sources folders?

Reply:
theres two in the docker file we 
COPY . /src/
WORKDIR /src
so it creates a new src file in the base directory and then moves everything in gearbox-matching into that folder

Reply:
we can get rid of it I think by removing this slash WORKDIR /src with WORKDIR src/

Reply:
or doing copy . .

Reply:
That is fine we can leave as is